app/client/src/components/pages/StateTribal.js

```diff
@@ -513,21 +518,28 @@ function StateTribal() {
                         secondClass = 'esri-menu__list-item-active';
                       }
 
+                      function handleClick(
+                        ev: React.KeyboardEvent | React.MouseEvent,
+                      ) {
+                        if (!isClick(ev)) return;
+
+                        setSelectedSource(source);
+                        setSourcesVisible(false);
+
+                        const searchInput =
+                          document.getElementById('hmw-search-input');
+                        if (searchInput) searchInput.focus();
+                      }
+
                       return (
                         <li
                           id={`source-${sourceIndex}`}
                           role="menuitem"
                           className={`esri-search__source esri-menu__list-item ${secondClass}`}
                           tabIndex="-1"
-                          key={`source-key-${sourceIndex}`}
-                          onClick={() => {
-                            setSelectedSource(source);
-                            setSourcesVisible(false);
-
-                            const searchInput =
-                              document.getElementById('hmw-search-input');
-                            if (searchInput) searchInput.focus();
-                          }}
+                          key={uuid()}
```

Comment:
I think the source could just be used here:
```suggestion
                          key={source}
```